TwilioVerify/TwilioVerify/Sources/Extensions/URLSession+Extensions.swift

```diff
@@ -34,11 +34,15 @@ extension URLSession {
         result(.failure(NetworkError.invalidResponse(errorResponse: data)))
         return
       }
+      Logger.shared.log(withLevel: .networking, message: "Response code: \(response.statusCode)")
       guard response.statusCode < 300 else {
         let failureResponse = FailureResponse(responseCode: response.statusCode, errorData: data, headers: response.allHeaderFields)
+        Logger.shared.log(withLevel: .networking, message: "Error body: \(String(data: data, encoding: .utf8))")
         result(.failure(NetworkError.failureStatusCode(failureResponse: failureResponse)))
         return
       }
+      Logger.shared.log(withLevel: .networking, message: "Response headers: \(response.allHeaderFields.map{"\($0): \($1)"}.joined(separator: ", "))")
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/db23e7504a6c641506761ed878f023492a1b05f0/TwilioVerify/TwilioVerify/Sources/Extensions/URLSession+Extensions.swift#L44"></span>Opening braces should be preceded by a single space and on the same line as the declaration.<br />
<code>opening_brace</code> <code>URLSession+Extensions.swift:44</code></td>
    </tr>
  </tbody>
</table>